packages/calypso-products/src/plans-list.tsx

```diff
@@ -1622,4 +1623,26 @@ PLANS_LIST[ PLAN_WPCOM_PRO ] = {
 		FEATURE_SIMPLE_PAYMENTS,
 		FEATURE_WORDADS_INSTANT,
 	],
+	getIncludedFeatures: () => [
+		FEATURE_ADVANCED_DESIGN,
+		FEATURE_ADVANCED_SEO_EXPANDED_ABBR,
+		FEATURE_AUDIO_UPLOADS,
+		FEATURE_CLOUDFLARE_ANALYTICS,
+		FEATURE_COLLECT_PAYMENTS_V2,
+		FEATURE_EARN_AD,
+		FEATURE_EMAIL_FORWARDING_EXTENDED_LIMIT,
+		FEATURE_GOOGLE_ANALYTICS,
+		FEATURE_GOOGLE_MY_BUSINESS,
+		FEATURE_HOSTING,
+		FEATURE_MEMBERSHIPS,
+		FEATURE_NO_BRANDING,
+		FEATURE_REPUBLICIZE,
+		FEATURE_PREMIUM_CONTENT_BLOCK,
+		FEATURE_SEO_PREVIEW_TOOLS,
+		FEATURE_SFTP_DATABASE,
+		FEATURE_SIMPLE_PAYMENTS,
```

Comment:
This one is already listed above so doesn't need to be added here (but there's presumably no harm either).